src/data/types.rs

```diff
@@ -281,113 +281,165 @@ pub(super) fn print_type(
     name: Option<InternedStr>,
     f: &mut Formatter,
 ) -> fmt::Result {
-    print_pre(ctype, f)?;
-    print_mid(ctype, name, f)?;
-    print_post(ctype, f)
+    print_declarator(ctype, name, f)?;
+
+    Ok(())
 }
```

Comment:
I think we should get rid of this and rename `print_declarator` to `print_type`.

Reply:
Forgot to do that